src/main/java/com/github/tomakehurst/wiremock/admin/Conversions.java

```diff
@@ -30,7 +30,9 @@ public static Integer toInt(QueryParameter parameter) {
 
   public static Date toDate(QueryParameter parameter) {
     try {
-      return parameter.isPresent() ? new ISO8601DateFormat().parse(parameter.firstValue()) : null;
+      return parameter.isPresent()
+          ? new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").parse(parameter.firstValue())
```

Comment:
Are we changing the contract here?

I think the class we're replacing will allow milliseconds whereas it looks like this won't.

Reply:
Ah fair point, I looked upon the usages and based myself upon the usage for ParseDateHelper which does not use ms. Do you prefer a singular contract or optional ms?